.gitpod.yml

```diff
@@ -0,0 +1,17 @@
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+  - name: Compile the protocol buffers & launch the backend with docker-compose
+    init: |
+      cd app
+      docker run --rm -w /app -v $(pwd):/app couchers/grpc ./generate_protos.sh
+      docker-compose up --build
+  - name: Install and launch the frontend with yarn
+    init: |
+      cd app/frontend 
+      yarn install
+    command: yarn start
```

Comment:
I guess this needs to be changed to something like `REACT_APP_API_BASE_URL=$(gp port 8888) yarn start`

Reply:
I've decided to leave out the automation for `yarn start` as I highlighted in [this comment](https://github.com/Couchers-org/couchers/pull/1749#discussion_r684790473).

So I guess you could run `REACT_APP_API_BASE_URL=$(gp port 8888) yarn start` from the terminal or change the file directly to use that API in `.env.development`